packages/clerk-js/src/utils/zxcvbn.ts

```diff
@@ -3,17 +3,32 @@ import type { ZxcvbnResult } from '@clerk/types';
 export type zxcvbnFN = (password: string, userInputs?: (string | number)[]) => ZxcvbnResult;
 
 export const loadZxcvbn = () => {
-  return Promise.all([import('@zxcvbn-ts/core'), import('@zxcvbn-ts/language-common')]).then(
-    ([coreModule, languageCommonModule]) => {
-      const { zxcvbnOptions, zxcvbn } = coreModule;
-      const { dictionary, adjacencyGraphs } = languageCommonModule;
-      zxcvbnOptions.setOptions({
-        dictionary: {
-          ...dictionary,
-        },
-        graphs: adjacencyGraphs,
-      });
-      return zxcvbn;
-    },
-  );
+  return loadLib().then(([coreModule, languageCommonModule]) => {
+    const { zxcvbnOptions, zxcvbn } = coreModule;
+    const { dictionary, adjacencyGraphs } = languageCommonModule;
+    zxcvbnOptions.setOptions({
+      dictionary: {
+        ...dictionary,
+      },
+      graphs: adjacencyGraphs,
+    });
+    return zxcvbn;
+  });
+};
+
+// We can only dynamically import zxcvbn in the browser.
+// On the headless variant we need to require it, and
+// this is because import() uses the document object
+// which is not available in Expo or other native environments.
+const loadLib = () => {
+  let core;
+  let languageCommon;
+  if (__IS_BROWSER__) {
+    core = import('@zxcvbn-ts/core');
```

Comment:
❓ Is this a **sync** import? 
Pretty sure this is not widely supported yet - this should probably be `await`ed here.

Reply:
Is there a webpack config we can leverage where dynamic imports are "ignored" and instead they are bundled as normal?

Reply:
> Is this a sync import?
This is followed by:
```
return Promise.all([core, languageCommon]);
```
so, we don't expect it to be sync.

Reply:
> Is there a webpack config we can leverage where dynamic imports are "ignored" and instead they are bundled as normal?

Didn't find anything. :(